### PR TITLE
feat(flag): adding flag documentation

### PR DESCRIPTION
**Pull request by lacymorrow**
`yarn`
`yarn start:website`

Visit: http://localhost:8000/components/flag/

@jayarnielsen I couldn't figure out an easy way to render an `<a href>` here (or a newline), I think it's fine for now

image

### CODE REVIEW

**Review by jayarnielsen — DISMISSED** [review]

**Comment by lacymorrow** [comment]
@jayarnielsen did you remove `"css.escape": "1.5.1"` and `"event-source-polyfill": "1.0.24"`?

Want to make sure I'm not re-adding them

File: website/package.json [hunk 1]
```diff
@@ -31,6 +31,9 @@
     "@twilio-labs/match-themes": "workspace:packages/match-themes",
     "@twilio-labs/match-tokens": "workspace:packages/match-tokens/core",
     "clsx": "1.1.1",
+    "css.escape": "1.5.1",
+    "event-source-polyfill": "1.0.24",
```
Comment (lacymorrow) [inline website/package.json:35]: @jayarnielsen These^^

Comment (jayarnielsen) [reply website/package.json:35]: Yes, you can remove those. They were installed because of a bug that has been resolved in the latest version of Gatsby

**Comment by lacymorrow** [comment]
@jayarnielsen any idea why the build is failing?

**Comment by jayarnielsen** [comment]
The `--immutable` install is failing which usually means that the lock file is in a bad state. Try deleting the yarn lack and reinstalling.

**Review by jayarnielsen — APPROVED** [review]

**Pull request merged**